numpy/core/src/umath/logical_gufuncs.c.src

```diff
@@ -598,6 +598,8 @@ void InitLogicalGufuncs(PyObject *dictionary,
     PyDict_SetItemString(dictionary, "@NAME@", f);
     Py_DECREF(f);
 
+    }; // close bracket surrounding inner repeat
```

Comment:
I'd probably argue for indenting this whole block, like you would any other set of curlies